activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/main/java/org/activiti/cloud/starter/rb/configuration/RuntimeBundleSwaggerConfig.java

```diff
@@ -52,9 +55,34 @@ public GroupedOpenApi runtimeBundleApi(@Value("${activiti.cloud.swagger.rb-base-
             .addOpenApiCustomizer(openApi ->
                 openApi.addExtension(BaseOpenApiBuilder.SERVICE_URL_PREFIX, swaggerBasePath)
             )
+            .addOpenApiCustomizer(openApiCustomizer())
             .build();
     }
 
+    public OpenApiCustomizer openApiCustomizer() {
+        return openAPI ->
+            openAPI
+                .getPaths()
+                .values()
+                .stream()
+                .flatMap(val -> val.readOperations().stream())
+                .flatMap(operation -> operation.getResponses().entrySet().stream())
+                .filter(entry -> entry.getKey().matches("200"))
+                .forEach(entry -> {
+                    Content contents = entry.getValue().getContent();
+                    String applicationHal = "application/hal+json";
```

Comment:
You can re-use Spring constants here to avoid any risk of typo:
- `MediaTypes.HAL_JSON_VALUE` from package `org.springframework.hateoas`
- `MediaType.APPLICATION_JSON_VALUE` from package `org.springframework.http`

Reply:
Done